app/models/merchant.rb

```diff
@@ -27,4 +28,8 @@ def distinct_cities
     item_orders.distinct.joins(:order).pluck(:city)
   end
 
+  def pending_orders
+    orders.distinct.where(status: 0)
+  end
```

Comment:
this groups item_orders into just orders

Reply:
nice looks good